src/containers/profile/profile.component.jsx

```diff
@@ -0,0 +1,37 @@
+import React from "react";
+import { auth } from "database";
+import { updateUser } from "database/updateUser";
+import { useSelector } from "react-redux";
+
+const Profile = (props) => {
+  const userFromState = useSelector((state) => state.users.user);
+
+  const updateUserInfo = () => {
+    const data = {
```

Comment:
Try naming variables more meaningfully, for example what does `data` actually hold so that reading/maintaining the code becomes easier?